calculator.rb

```diff
@@ -0,0 +1,34 @@
+def prompt_operator
+  puts "enter a math operation"
+  operator = gets.chomp
+end
+
+def prompt_number
+  begin
+    puts "enter a number for the math operation"
+    Float(gets.chomp) 
```

Comment:
I probably would prefer an explicit return here, because I am that person sorry! :sob: